pydarn/plotting/rtp.py

```diff
@@ -109,9 +110,13 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
         yspacing: int
             sets the spacing between ticks
             Default: 200
-        coord: Coord
+        coord: Coords
             set the y-axis to a desired coordinate system
-            Default: Coord.SLANT_RANGE
+            Default: Coords.SLANT_RANGE
+        hgt: int
```

Comment:
``` suggestion
        reflection_height: int
```

Reply:
So later on in summary plots, you have this value as a  float? Should this be a float?